app/lib/staypuft/seeder.rb

```diff
@@ -342,7 +342,7 @@ def functional_dependencies
       ceph_cluster_network      = { :string => "<%= @host.network_query.network_address_for_host('#{Staypuft::SubnetType::STORAGE_CLUSTERING}') %>" }
       # FIXME: this should actually be STORAGE instead of PXE, but only after we have a reliable way of identifying DNS names
       #        on the storage network
```

Comment:
Also at  lines 351-352 there's another related FIXME comment about the storage network -- lets delete that at the same time since it's no longer relevant here.